src/components/sync-connected-sites.tsx

```diff
@@ -237,12 +274,14 @@ const SyncConnectedSiteControls = ( {
 						localSite={ selectedSite }
 						remoteSite={ connectedSite }
 						onSubmit={ ( tree ) => {
-							const optionsToSync = convertTreeToOptionsToSync( tree );
+							const syncOptions = convertTreeToOptionsToSync( tree );
 
 							if ( syncDialogType === 'push' ) {
-								void pushSite( connectedSite, selectedSite, { optionsToSync } );
+								void pushSite( connectedSite, selectedSite, syncOptions );
 							} else {
-								pullSite( connectedSite, selectedSite, { optionsToSync } );
+								pullSite( connectedSite, selectedSite, {
+									optionsToSync: syncOptions.optionsToSync,
+								} );
```

Comment:
Not critical, but for consistency, I would keep it this way. WDYT?
```suggestion
								pullSite( connectedSite, selectedSite, syncOptions );
```